src/pages/guides/index.md

```diff
@@ -49,4 +55,17 @@ services:
   - code: CampaignStandard
 extension:
   serviceCode: dx/excshell/1
+runtime: true
```

Comment:
`runtime` and `event` are marked as future improvements in this document: https://wiki.corp.adobe.com/pages/viewpage.action?spaceKey=AdobeCloudPlatform&title=App+Builder+Template+Install+Configuration.

I think they should be out of scope for now. @slitviachenko thoughts?

Reply:
@roribio , I see in the "Specification" section that only "hooks" is marked as "Future", "runtime" and "event" are new properties. Shaz also mentioned those changes in [his comment](https://adobedx.slack.com/archives/C02UWB27XJN/p1653487808846049). So I think we should support them.